libs/frc20_abi/src/lib.sw

```diff
@@ -0,0 +1,22 @@
+library frc20_abi;
+
+use std::u256::U256;
+
+abi Token {
+    #[storage(read)]
+    /// Get the total supply of the token.
+    /// MUST track all mint and burn operations.
+    /// MAY store the amount as another type (i.e: 'u64') internally.
+    fn total_supply() -> U256;
+
+    /// Get the name of the token
+    /// Example (with trailing padding): "MY_TOKEN                                                        "
+    fn name() -> str[64];
```

Comment:
Bytes support is gonna hit the Rust SDK soon. I'd use that type instead or perhaps the String type from this repo. cc @bitzoic ?

Reply:
Pretty sure `String`s can't be stored.

Reply:
I think we all decided fixed size strings is perfectly fine for now. Just pad out the name with trailing spaces and any front-end will know to trim it